packages/jaeger-ui/src/components/TracePage/url/index.tsx

```diff
@@ -12,19 +12,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import queryString from 'query-string';
+
 import prefixUrl from '../../../utils/prefix-url';
 
 import { TNil } from '../../../types';
 
 export const ROUTE_PATH = prefixUrl('/trace/:id');
 
-export function getUrl(id: string) {
-  return prefixUrl(`/trace/${id}`);
+export function getUrl(id: string, uiFind?: string): string {
```

Comment:
Not very familiar with tsx syntax, is "uiFind?" expected with "?"

Reply:
the `?` just means it's optional. not all trace urls are deep links.